filecoin-proofs/tests/api.rs

```diff
@@ -375,8 +373,11 @@ fn window_post<Tree: 'static + MerkleTreeTrait>(
     /////////////////////////////////////////////
     // The following methods of proof generation are functionally equivalent:
     // 1)
-    //let proof = generate_window_post::<Tree>(&config, &randomness, &priv_replicas, prover_id)?;
-    //
+    let proof = generate_window_post::<Tree>(&config, &randomness, &priv_replicas, prover_id)?;
```

Comment:
Commit description is misleading (mentioning for anyone else trying to make sense of it). It's Window PoSt that is affected.

Reply:
No need.